back-end/api/quickstart/tests/endpoints/test_inbox.py

```diff
@@ -12,7 +12,7 @@ class GetInbox(TestCase):
   def setUp(self):
     self.inbox = Inbox.objects.create(author="testAuthorId")
     self.inbox.items.append(get_test_post_fields())
-    self.inbox.items.append(get_test_follow_fields())
+    self.inbox.items.append(get_sender_fields())
```

Comment:
This should be get_friend_request_fields(), we'll most likely have to rename this in the future and add this new method.